dotcom-rendering/src/web/components/DynamicSlowMPU.tsx

```diff
@@ -125,26 +132,12 @@ const ColumnOfThree50_Ad50 = ({
 	);
 };
 
-/**
- * DynamicSlowMPU
- *
- * This container only allows big and standard cards (from groupedTrails)
- *
- * Layout is dynamic deopending on the number of big cards. You're only
- * allowed to have 2 or 3 big cards. If you pass 1 a standard card will
- * get promoted to make two bigs. If you pass more than 3 bigs then the
- * extras will get demoted to standard.
- *
- * If you pass no bigs at all the top slice will not render and a special
- * 3 column layout is used for the remaining slice.
- *
- */
-export const DynamicSlowMPU = ({
+const RenderMPU = ({
```

Comment:
Is there a reason we call these `Render{ComponentName}`? The pattern elsewhere is to just use the `{ComponentName}` and rendering if inferred from it being a component.

Reply:
Given these aren't exported, so namespaced to this component, we could just name them `MPU / NonMPU`.

Taking this a little further, could we just render this in the `DynamicSlowMPU` component given they are never reused?

Reply:
I'm happy with a different naming (perhaps `MPUSlice` vs. `NonMPUSlice`?)

But I think there's value to pulling them out as separate components has value even without reuse here. We've essentially got two totally different layouts depending on whether `renderAds` is true or not. Having each of the layouts in its own component/function makes it clear that this is the main thing determining how the container gets rendered. It also means that it's easier to determine which layout you're changing in the future (at least for me I find that identifying where you are in a ternary expression is quite difficult when the two sub-expressions either side of the colon are long, nested expressions themselves.)

Reply:
Renaming makes sense 👍 

I'm also still inclined to keep the two as separate components, particularly as one contains a couple of switch statements - this feels like it splits up the different steps in the logic effectively.